### DISCUSSION TITLE
Welcome to Telemeta Discussions!

### DISCUSSION THREAD

**Discussion by yomguy** [Parisson/Telemeta#231] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by josimonnot** [comment]
Josephine Simonnot, Ingénieur de recherche, CNRS, INSHS. I was responsible for the CREM Telemeta and LAM Telemeta. I wish to continue the development of Timeside for research audio data, related to the existing infrastructures.
